### PR TITLE
Simplify expression for tangent in hyperelasticity tutorial

### PR DESCRIPTION
**Pull request by KnutAM**
* Simplify expression for tangent
* Use new admonition syntax
* Add test for simplified equation

### CODE REVIEW

File: docs/src/literate-tutorials/hyperelasticity.jl [hunk 1]
```diff
@@ -99,60 +99,51 @@ using Ferrite, Tensors, TimerOutputs, ProgressMeter, IterativeSolvers
...
-# : \mathbf{F}^\mathrm{T} \bar{\otimes} \mathbf{I},
-# \end{aligned}
-# ```
-# where we used the fact that ``\mathbf{S}`` is symmetric (``S_{lj} = S_{jl}``) and that
-# ``\frac{\partial \mathbf{S}}{\partial \mathbf{C}}`` is *minor* symmetric (``\frac{\partial
-# S_{mq}}{\partial C_{no}} = \frac{\partial S_{mq}}{\partial C_{on}}``).
-#md # ```@raw html
-#md # </div></details>
-#md # ```
+# !!! details "Derivation of $\partial \mathbf{P} / \partial \mathbf{F}$"
```
Comment (fredrikekre) [inline docs/src/literate-tutorials/hyperelasticity.jl:107]: This will only work for Documenters HTML output, not the notebooks (xref https://fredrikekre.github.io/Literate.jl/v2/tips/#admonitions-compatibility)

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl:107]: Oh, is that the same above from #846 ? 
Or is it just the math in the header?

Comment (fredrikekre) [reply docs/src/literate-tutorials/hyperelasticity.jl:107]: Yea, see https://nbviewer.org/github/Ferrite-FEM/Ferrite.jl/blob/gh-pages/dev/tutorials/hyperelasticity.ipynb (forgot about this while reviewing that PR)

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl:107]: Yes, realized too (also that the link in the docs hasn't been updated since the "example" time...)
What do you think about introducing the preprocessing commands suggested in the Literate docs?
Would be quite nice to be able to use admonitions in the examples.

Comment (termi-official) [reply docs/src/literate-tutorials/hyperelasticity.jl:107]: The citation is also broken x.x

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl]: Should it be OK now after https://github.com/fredrikekre/Literate.jl/pull/259?

Comment (termi-official) [reply docs/src/literate-tutorials/hyperelasticity.jl]: I would just merge and look. Should be fine I think.

Comment (termi-official) [reply docs/src/literate-tutorials/hyperelasticity.jl:109]: Can we also quickly leave some links to material with an ELI5 for index notation and the used symbols here?

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl:109]: Added link to my [website](https://knutam.github.io/tensors/Theory/IndexNotation/), hope that is ok?

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl:109]: Well, the line break broke the notebook view, but looks correct in markdown. Bug in Literate @fredrikekre ? 



<details>

```
{
   "cell_type": "markdown",
   "source": [
    "> **Derivation of $\\partial \\mathbf{P} / \\partial \\mathbf{F}$**\n",
    ">\n",
    "> *Tip:* See [knutam.github.io/tensors](https://knutam.github.io/tensors/Theory/IndexNotation/) for\n",
    "> an explanation of the index notation used in this derivation\n",
    "\n",
    "    Using the product rule, the chain rule, and the relations $\\mathbf{P} = \\mathbf{F} \\cdot\n",
    "    \\mathbf{S}$ and $\\mathbf{C} = \\mathbf{F}^\\mathrm{T} \\cdot \\mathbf{F}$, we obtain the\n",
    "    following:\n",
    "    $$\n",
    "    \\begin{aligned}\n",
    "    \\frac{\\partial P_{ij}}{\\partial F_{kl}} &=\n",
    "    \\frac{\\partial (F_{im}S_{mj})}{\\partial F_{kl}} \\\\ &=\n",
    "    \\frac{\\partial F_{im}}{\\partial F_{kl}}S_{mj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}\\delta_{ml} S_{mj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\\frac{\\partial C_{no}}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}S_{lj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    \\frac{\\partial (F^\\mathrm{T}_{np}F_{po})}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}S^\\mathrm{T}_{jl} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    \\left(\n",
    "    \\frac{\\partial F^\\mathrm{T}_{np}}{\\partial F_{kl}}F_{po} +\n",
    "    F^\\mathrm{T}_{np}\\frac{\\partial F_{po}}{\\partial F_{kl}}\n",
    "    \\right) \\\\ &=\n",
    "    \\delta_{ik}S_{jl} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    (\\delta_{nl} \\delta_{pk} F_{po} + F^\\mathrm{T}_{np}\\delta_{pk} \\delta_{ol}) \\\\ &=\n",
    "    \\delta_{ik}S_{lj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    (F^\\mathrm{T}_{ok} \\delta_{nl} + F^\\mathrm{T}_{nk} \\delta_{ol}) \\\\ &=\n",
    "    \\delta_{ik}S_{jl} +\n",
    "    2\\, F_{im} \\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    F^\\mathrm{T}_{nk} \\delta_{ol} \\\\\n",
    "    \\frac{\\partial \\mathbf{P}}{\\partial \\mathbf{F}} &=\n",
    "    \\mathbf{I}\\bar{\\otimes}\\mathbf{S} +\n",
    "    2\\, \\mathbf{F} \\cdot \\frac{\\partial \\mathbf{S}}{\\partial \\mathbf{C}}\n",
    "    : \\mathbf{F}^\\mathrm{T} \\bar{\\otimes} \\mathbf{I},\n",
    "    \\end{aligned}\n",
    "    $$\n",
    "    where we used the fact that $\\mathbf{S}$ is symmetric ($S_{lj} = S_{jl}$) and that\n",
    "    $\\frac{\\partial \\mathbf{S}}{\\partial \\mathbf{C}}$ is *minor* symmetric ($\\frac{\\partial\n",
    "    S_{mj}}{\\partial C_{no}} = \\frac{\\partial S_{mj}}{\\partial C_{on}}$)."
   ],
   "metadata": {}
  },
```

</details>

Comment (KnutAM) [reply docs/src/literate-tutorials/hyperelasticity.jl:109]: But should be fixed for now by adding the workaround to remove the line break from the notebook.
